src/config.ts

```diff
@@ -16,13 +19,22 @@ export type ConfigAliasOptions = {
   defaultHidden: boolean;
 };
 
-const defaultConfig: Config = {
+export type InternalConfig = Config & {
+  /** Whether to allow RNTL to use latest and greatest improvements even if they are breaking changes. */
+  allowBreakingChanges: boolean;
```

Comment:
since it's an internal option now, I would vote for `useNextMajorVersionFeatures` or something like that, or `useLatestApis` / `useLatestFeatures` like you suggested

Reply:
`moveFastAndBreakThings`, `livingOnTheEdge` ;-)

Reply:
how about slight change to `useBreakingChanges`? `useNextMajorVersionFeatures` seems a bit longish. Also thought of `useVersionNext` `useVersionNextApis`, but somehow "breaking changes" stuck as most precise. Wdyt @MattAgn?

Reply:
Sounds good to me :)